scripts/add_ts_data_bidmatrixinfo.py

```diff
@@ -0,0 +1,154 @@
+###this script adds time series data to the bid matrix information objects in the powerops client
+
+from typing import Union
+
+from cognite.client.data_classes import TimeSeries, TimeSeriesList
+from cognite.client.exceptions import CogniteNotFoundError
+from cognite.client import CogniteClient
+from cognite.powerops import PowerOpsClient
+
+from cognite.powerops.client._generated.v1.data_classes import BidMatrixInformationWrite
+import random
+import arrow
+
+
+def create_time_series_objects():
+    ts_list = [
+        TimeSeries(external_id="emilie_ts_onhour", name="emilie_ts_onhour", metadata={"data_type": "on_hour"}),
+        TimeSeries(external_id="emilie_ts_empty", name="emilie_ts_empty", metadata={"data_type": "none"}),
+        TimeSeries(external_id="emilie_ts_offhour", name="emilie_ts_offhour", metadata={"data_type": "off_hour"}),
+    ]
+    return ts_list
+
+
+# Function to get existing time series from CDF
+def get_existing_time_series(
+    client: CogniteClient, ts_list: list[TimeSeries]
+) -> tuple[list[TimeSeries], list[TimeSeries]]:
```

Comment:
**note (for-future-reference):** the standard practice according the the [PEP 8 guidelines](https://peps.python.org/pep-0008/#documentation-strings) is to write comments documenting what a function does in a docstring inside the function at the start

```suggestion
def get_existing_time_series(
    client: CogniteClient, ts_list: list[TimeSeries]
) -> tuple[list[TimeSeries], list[TimeSeries]]:
"""
Fetches time series that exist in CDF based on a list of input time series to search for.

Args:
    client (CogniteClient): An authenticated CDF client.
    ts_list (list[TimeSeries]): A list of time series to search for.

Returns:
    existing_ts (list[TimeSeries]): A list of the time series that already exist in CDF based on the input list.
    missing_ts (list[TimeSeries]): A list of the time series that do not exist in CDF.
"""
```